app/controllers/api/v1/customers_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Api::V1::CustomersController < ApplicationController
+
+  def index
+    render json: Customer.all, only: [:id, :first_name, :last_name]
```

Comment:
i'd suggest using a Serializer here rather than just specifying the data right in the controller. Using a serializer - either [ActiveModelSerializers](https://github.com/rails-api/active_model_serializers) or [Jbuilder](https://github.com/rails/jbuilder) will allow you to specify what attributes should be shown to the outside world and then you can reuse that in other files 👍